palak/src/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+// import { Children } from "react";
+// import withLoading from "./withLoading";
+const Button = ({ onClick, className, children }) => {
+  return (
+    <button
+      className={`text-sm font-medium h-11 rounded-md w-full focus-visible:outline-none focus-visible:ring-2  max-w-sm mt-10 text-white bg-blue-800 hover:bg-blue-700 ${className}`}
+      onClick={onClick}
+    >
+      {children}
+    </button>
+    // <div>
+    //   {isLoading ? (
+    //     <div className="flex opacity-40 flex-row">
+    //       <div>
+    //         <button
+    //           className={`text-sm font-medium h-11 rounded-md w-full focus-visible:outline-none focus-visible:ring-2  max-w-sm mt-10 text-white bg-blue-800 hover:bg-blue-700 ${className}`}
```

Comment:
If some code is commented out and is not required for review or for some other references, then you can delete that.